hugr-core/src/extension/declarative/types.rs

```diff
@@ -119,52 +116,9 @@ impl TypeParamDeclaration {
     /// TODO: The parameter description is currently ignored.
```

Comment:
The parameter description no longer exists, so drop the comment.

Or reinstate the description - it's harmless, could sit there as a reminder what the string is actually supposed to be, but I absolutely do *not* insist